web/libs/core/src/pages/AccountSettings/sections/PersonalJWTToken.tsx

```diff
@@ -106,7 +106,15 @@ export function PersonalJWTToken() {
 
   const revoke = useCallback(
     async (token: string) => {
-      await revokeToken.mutateAsync({ token });
+      confirm({
+        title: "Revoke Token",
+        body: "Are you sure you want to delete this access token? Any application using this token will need a new token to be able to access Label Studio",
```

Comment:
for white labeling support
```suggestion
        body: `Are you sure you want to delete this access token? Any application using this token will need a new token to be able to access ${window?.APP_SETTINGS?.app_name || "Label Studio"}`,
```

Reply:
since there is no override in LSE - this will be needed